safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -116,6 +117,68 @@ def submit_signatures(self, safe_tx_hash: bytes) -> bool:
                 )
         return False
 
+    def batch_safe_txs(self, safe_nonce: int, safe_txs: [SafeTx]) -> bool:
```

Comment:
Why 2 different `batch_safe_txs` functions? Process for batching all the transactions before sending should be the same both for `tx-service` mode and `blockchain` mode. Logic for sending them to either the service or blockchain should be on the `drain` function

Reply:
Make sense